dask/array/core.py

```diff
@@ -1080,6 +1083,50 @@ def numblocks(self):
     def npartitions(self):
         return reduce(mul, self.numblocks, 1)
 
+    def compute_chunk_sizes(self):
+        """
+        Compute the chunk sizes for a Dask array. This is especially useful
+        when the chunk sizes are unknown (e.g., with conditional indexing
+        on Dask Array).
+
+        Notes
+        -----
+        This function modifies the Dask array in-place.
+
+        Examples
+        --------
+
+        >>> import dask.array as da
```

Comment:
Do the other examples have these imports?

Reply:
Yup: https://github.com/dask/dask/blob/973c6e1b2e38c2d9d6e8c75fb9b4ab7a0d07e6a7/dask/array/core.py#L439

When I don't have these imports doctest complains without a `# doctest: SKIP` flag.